src/components/App.tsx

```diff
@@ -251,10 +254,11 @@ export class App extends React.Component<AppProps, AppState> {
       this.toggleWorkspaceSplit();
     }
     if (!this.state.accountId) {
-      const createAccountResponse = await createAccount();
-      const key = createAccountResponse["key"];
-      this.state.keyStore.setKey(createAccountResponse["account_id"], key);
-      App.setAccountId(createAccountResponse["account_id"]);
+      const randomSuffix = Math.floor(Math.random() * 9999999999); 
```

Comment:
1. should we better just use public key suffix?
2. I feel like it makes sense to have "create account with random key and return account id" logic packaged into Nearlib, as it looks like would be common when testing.

Reply:
Public key would make the name too long. (We have a length limit currently)